src/lib/render.ts

```diff
@@ -1,166 +1,179 @@
-import { okhsv_to_srgb, srgb_to_okhsv } from "./colorconversion"
-import { picker_size, slider_width } from "./constants"
-import { Hsluv } from 'hsluv'
+import { useMode, modeRgb, modeOkhsl, modeOkhsv } from 'culori/fn'
+import { picker_size, slider_width } from './constants'
 
 const lowres_picker_size = Math.round((picker_size + 1) / 2)
 const picker_size_inv = 1 / picker_size
 
-const hsluv = new Hsluv()
+useMode(modeOkhsl)
+useMode(modeOkhsv)
```

Comment:
The only weird quirk about Culori is it is “side-effect-y” so loading colorspaces like this is necessary in order to use `{ mode: 'okhsl' … }` and `{ mode: 'okhsv' … }` below